examples/joy-ui-nextjs/src/components/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { extendTheme } from '@mui/joy/styles';
+import { Inter, Source_Code_Pro } from 'next/font/google';
+
+const inter = Inter({
+  subsets: ['latin'],
+  adjustFontFallback: false, // prevent automatically adding Arial as the fallback
+  fallback: ['var(--joy-fontFamily-fallback)'], // explicitly stated we wish to use Joy UI's fallback
+});
+
+const sourceCodePro = Source_Code_Pro({
+  subsets: ['latin'],
+  adjustFontFallback: false,
+  // copied from the default theme
+  fallback: [
+    'ui-monospace',
+    'SFMono-Regular',
+    'Menlo',
+    'Monaco',
+    'Consolas',
+    'Liberation Mono',
+    'Courier New',
+    'monospace',
+  ],
```

Comment:
We could get it with `extendTheme().fontFamily.code` no?

https://codesandbox.io/s/hungry-meninsky-jrq99h?file=/Demo.tsx:80-115

Something like?

```suggestion
  fallback: extendTheme().fontFamily.code,
```

Reply:
Tried 
```ts
fallback: extendTheme().fontFamily.code.split(","),
```

And get an error of:
```
  x Font loader values must be explicitly written literals.
```

It seems that it has to be a hardcoded value.

Reply:
What the origin of this error?

Reply:
It's from Next.js
<img width="990" alt="image" src="https://github.com/mui/material-ui/assets/18292247/099eac0f-86ef-4986-bbfa-affce17c847e">